python/pyspark/pandas/missing/frame.py

```diff
@@ -29,7 +29,7 @@ def _unsupported_property(property_name, deprecated=False, reason=""):
     )
 
 
-class _MissingPandasLikeDataFrame:
+class MissingPandasLikeDataFrame:
```

Comment:
Is this convention not used to 'hide' classes from docs or APIs in Python? I don't know if it applies to classes. Just wondering if it's intentional

Reply:
they will not be shown in the API references, because they are not registered in `python/docs/source/reference/pyspark.pandas`

All internal classes in Pandas-API-on-Spark but the three `_MissingPandasXXX` ones do not start with `_`, so I think it's fine to make this change.